api/openapi-spec/v3/api__v1_openapi.json

```diff
@@ -4199,6 +4199,10 @@
             "description": "storageClassName is the name of the StorageClass required by the claim. More info: https://kubernetes.io/docs/concepts/storage/persistent-volumes#class-1",
             "type": "string"
           },
+          "volumeAttributesClassName": {
+            "description": "volumeAttributesClassName is the name of the VolumeAttributesClass required by the claim. If specified, the provisioner will create or update the volume with the attributes defined in the corresponding VolumeAttributesClass. Different with storageClassName, it can be changed after the claim is created but an empty string value is disallowed. More info: https://kubernetes.io/docs/concepts/storage/persistent-volumes#volumeattributesclass (Alpha) Using this field requires the VolumeAttributesClass feature gate to be enabled.",
```

Comment:
Which povisioner you are referring here? The update is happening in the external-resizer not in the external-provisioner?

Reply:
How about `csi-driver` ?

Reply:
csi-driver makes sense